lib/solr/query/request.rb

```diff
@@ -0,0 +1,49 @@
+require 'solr/testing'
```

Comment:
I was actually thinking that this gem consumer should be requiring this when needed (in specs) and this require would re-open solr module and add a class method with last request details, but that's not really important now, let's change later.